### ISSUE TITLE
Callbacks PickedUp does not work

### ISSUE THREAD

**Issue by eXeGOODER** [ikkentim/SampSharp-streamer#15]
Initialize:

``` C#
PickedUp += CPickup_PickedUp;
```

Callbacks:

``` C#
// heritable class  : DynamicPickup

private void CPickup_PickedUp(object sender, PlayerEventArgs e)
{
 // doesn't work
}

public override void OnPickedUp(PlayerEventArgs e)
{
 // doen't work
}
```

Pickup types tested: 1, 21, 23

**Comment by ikkentim** [comment]
What version are you using?

**Comment by eXeGOODER** [comment]
SampSharp 0.6.2
SampSharp-Streamer 0.6.0
Streamer 2.7.9

**Comment by ikkentim** [comment]
I think it might be fixed in 0.7. I'll check when I'm back from my vacation
